src/constants.rs

```diff
@@ -310,21 +310,108 @@ impl Default for Viewtype {
     }
 }
 
+pub const DC_JOB_DELETE_MSG_ON_IMAP: i32 = 110;
+
+#[derive(Debug, Clone, Copy, PartialEq, Eq, FromPrimitive, ToPrimitive)]
+#[repr(u8)]
+pub enum KeyType {
+    Public = 0,
+    Private = 1,
+}
+
```

Comment:
diff looks weird here, this block was just moved up as the "test" block was somewhere in the middle of the file.